spec/features/notes_spec.rb

```diff
@@ -55,4 +55,19 @@
 
   end
 
+  describe "on a disabled story" do
+    let!(:story) do
```

Comment:
You can do that inside the `it` directly since we don't use the variable `story`. Besides that everything looks ok to me.